docs/development/reviews.md

```diff
@@ -0,0 +1,39 @@
+Some notes on how we do reviews
+===============================
+
+The Synapse team works off a shared review queue -- any new pull requests for
+Synapse (or related projects) has a review requested from the entire team. Team
+members should process this queue using the following rules:
+
+* Any high urgency pull requests (e.g. fixes for broken continuous integration
+  or fixes for release blockers);
+* Follow-up reviews for pull requests which have previously received reviews;
+* Any remaining pull requests.
+
+For the latter two categories above, older pull requests should be prioritised.
+
+It is explicit that there is no priority given to pull requests from the team
+(vs from the community).  If a pull request requires a quick turn around, please
+explicitly communiate this via [#synapse-dev:matrix.org](https://matrix.to/#/#synapse-dev:matrix.org)
+or as a comment on the pull request.
+
+Once an initial review has been completed and the author has made additional changes,
+follow-up reviews should go back to the same reviewer. This helps build a shared
+context and conversation between author and reviewer.
+
+As a team we aim to keep the number of inflight pull requests to a minimum to ensure
+that ongoing work is finished before starting new work.
+
+Performing a review
+-------------------
+
+Team members should do the following when reviewing:
+
+* Assigning themselves to the pull request;
+* Review the pull request;
```

Comment:
This feels a bit awkward: "Team members should review the pull request when reviewing"

Reply:
I didn't really want to get into "what to look for in a review" in this PR (since there are other conversations happening about that), so it is a bit terse.

Pretty much I want to say "now do the actual review".

Reply:
I reworded the section a bit, I hope it makes more sense now.